src/main/java/seedu/linkedout/logic/parser/ParserUtil.java

```diff
@@ -66,18 +67,33 @@ public static Phone parsePhone(String phone) throws ParseException {
     }
 
     /**
-     * Parses a {@code String linkedout} into an {@code Address}.
+     * Parses a {@code String linkedout} into an {@code Job}.
      * Leading and trailing whitespaces will be trimmed.
      *
      * @throws ParseException if the given {@code linkedout} is invalid.
```

Comment:
Should the comments be changed? The rest of the file has changed comments so this might reflect some inconsistency.

I believe `parseStage` a few lines down has the same issue.
```suggestion
     * @throws ParseException if the given {@code job} is invalid.
```

Reply:
a972860c86657043b96a8bc0fbd3c3823566ffc4